metadata.json

```diff
@@ -5,14 +5,25 @@
   "summary": "A helper for writing Bolt plugins in Ruby",
   "license": "Apache-2.0",
   "source": "https://github.com/puppetlabs/puppetlabs-ruby_plugin_helper",
-  "dependencies": [],
+  "dependencies": [
+
+  ],
+  "operatingsystem_support": [
+    "RedHat based Linux",
+    "Debian based Linux",
+    "Windows"
```

Comment:
dont think this is valid

Reply:
maybe @david22swan can confirm

Reply:
yeah seems it... 🤔 https://github.com/puppetlabs/pdk/blob/main/lib/pdk/module/metadata.rb#L71C1-L75C15

Reply:
Don't mind me then!